packages/@guardian/source-react-components-development-kitchen/src/lines/StraightLines.tsx

```diff
@@ -0,0 +1,52 @@
+import type { SerializedStyles } from '@emotion/react';
+import type { EmotionJSX } from '@emotion/react/types/jsx-namespace';
+import { breakpoints, space } from '@guardian/source-foundations';
+import type { LineCount } from './Lines';
+
+const thickness = 1;
+const gapVertical = space[1];
+const maxWidth = breakpoints.wide;
+
+const getHeight = (count: LineCount): number =>
+	gapVertical * (count - 1) + thickness;
+
+export const StraightLines = ({
+	count,
+	color,
+	cssOverrides,
+}: {
+	count: LineCount;
+	color: string;
+	cssOverrides?: SerializedStyles | SerializedStyles[];
+}): EmotionJSX.Element => {
+	const height = getHeight(count);
+	const viewBox = `0 0 ${maxWidth} ${height}`;
+
+	const lines = [];
+	for (let index = 0; index < count; index++) {
+		lines.push(
+			<line
+				key={index}
+				x1={1}
```

Comment:
Not `0`? Is this because the thickness of the line would result in the viewbox cropping the edge?

Reply:
```suggestion
				x1={0}
```